src/game.h

```diff
@@ -411,7 +411,7 @@ class game
 
         void open_gate( const int examx, const int examy, const ter_id handle_type );
 
-        bionic_id random_good_bionic() const; // returns a non-faulty, valid bionic
+        std::string random_good_bionic() const; // returns a non-faulty, valid bionic
```

Comment:
This seems to be the only bionic function in the game class, it should probably be moved into bionics.h